pyproject.toml

```diff
@@ -156,7 +149,14 @@ matrix.airflow.dependencies = [
 freeze = "pip freeze"
 test = 'pytest -vv --durations=0 . -m "not integration"'
 test-cov = 'pytest -vv --cov=cosmos --cov-report=term-missing --cov-report=xml --durations=0 -m "not integration"'
-test-integration = "pytest -vv --cov=cosmos --cov-report=term-missing --cov-report=xml --durations=0 -m integration"
+# we install using the following workaround to overcome installation conflicts, such as:
+# apache-airflow 2.3.0 and dbt-core [0.13.0 - 1.5.2] and jinja2>=3.0.0 because these package versions have conflicting dependencies
+test-integration = """\
+pip uninstall dbt-postgres; \
+rm -rf airflow.*; \
+airflow db init; \
+pip install dbt-postgres; \
```

Comment:
does this mean every time we run the tests, we're installing pypi packages and re-initializing the airflow db? is there any better way to do this? just worried this makes the local iteration speed much slower because we're not really re-using an environment

Reply:
Yes, @jlaneve - and that's what we're doing in the CI now.

Given the conflict errors between `dbt` & `airflow,` I didn't manage to put the `dbt-postgres` dependency together with the other `pyproject.toml` dependencies.

I can happily remove these lines. Would you prefer that?
```
rm -rf airflow.*; \
airflow db init; \
```

The only consequence is that the developer doing the local setup will have to manage their Airflow Home and avoid conflicts if they are running multiple versions of Airflow.

Reply:
ah interesting - maybe we break it out into two separate scripts? one for `init-db` and one for `run-integration-tests`

Reply:
Good idea, @jlaneve , I'll break this up early tomorrow BST!

Reply:
@jlaneve please, check if e4420083235f4c31cbcc6733a0ed45b3b6b84ab5 is okay!